.github/workflows/test.yml

```diff
@@ -335,12 +335,14 @@ jobs:
       #########################################################################
       # COVERAGE CHECK BACKEND COMMUNITY-SERVER ####################################
       ##########################################################################
-      #- name: backend community simplecov | Coverage check
+      - name: backend community | Coverage check
+        run: |
+          docker run -v ~/coverage:/var/www/cakephp/webroot/coverage gradido/community_server:test vendor/bin/coverage-check ./webroot/coverage/clover.xml 14
```

Comment:
Please make this an external check. The `coverage-check-action` creates a new workflow check - we should do that analog. So its all fine, but the coverage check must be its own - do not do it in `unit_test_community_server`. It migth be nessecary to upload the coverage clover.xml to github

Reply:
![image](https://user-images.githubusercontent.com/1238238/121509064-8d95d500-c9e6-11eb-9b16-cc290ae95f44.png)

Reply:
I have modified the action to also understand the coverage report from phpunit in txt format. 
I have never coded with ruby but it was not so hard :)

I think it was easier as the other way round, to try to make a own github action with the phpunit clover.xml interpretation class.